array.c

```diff
@@ -3431,26 +3510,49 @@ sort_2(const void *ap, const void *bp, void *dummy)
 
 /*
  *  call-seq:
- *     ary.sort!                   -> ary
- *     ary.sort! {|a, b| block}    -> ary
+ *    array.sort! -> self
+ *    array.sort! {|a, b| ... } -> self
+ *
+ *  Returns +self+ with its elements sorted.
```

Comment:
I would like to keep the "in place" here like "… with its elements sorted in place" like you have for #rotate!

Reply:
Done.